docs/source/examples.rst

```diff
@@ -6,22 +6,6 @@ Usage Examples
 .. toctree::
    :maxdepth: 1
 
-.. _example_dependencies:
-
-Example Dependencies
---------------------
-
-Some examples use additional dependencies, which are listed in `examples_requirements.txt <https://github.com/lanl/scico/blob/main/examples/examples_requirements.txt>`_.
-The additional requirements should be installed via pip, with the exception of ``astra-toolbox``,
-which should be installed via conda:
-
-   ::
-
-      conda install -c astra-toolbox astra-toolbox
-      pip install -r examples/examples_requirements.txt # Installs other example requirements
-
-The dependencies can also be installed individually as required.
-
```

Comment:
Did you remove the example related dependencies? I don't think these are there anywhere else.

Reply:
This is going to be a continual pain until #195 is addressed. I'll fix manually for now.